src/routes/__tests__/routes.test.js

```diff
@@ -0,0 +1,7 @@
+import indexRoutes from '..';
+
+describe('Routes tests', () => {
+  it('should be an object of routs', () => {
+    expect(indexRoutes).toBeInstanceOf(Object);
```

Comment:
this does not really prove that it is a route object. can you assert the properties of the object?  have a look at jest `toHaveProperty` method

[link to method](https://jestjs.io/docs/en/expect#tohavepropertykeypath-value)

Reply:
Thanks for the review. The link was very helpful. I have updated the test using `toBeInstanceOf`.